### ISSUE TITLE
Containers whose leafs are set to default values are no longer suppressed in get-config response beginning in 6ab0617

### ISSUE THREAD

**Issue by alangefe** [CESNET/Netopeer2#316]
Beginning in 6ab0617, the server is no longer suppressing containers whose only leafs are set to default values when it presents the configuration via a `get-config` response.

For example, given the following model:

```
container test-container {
  leaf test-leaf {
    type string;
    default "foo";
  }
}
```

and assuming the node `test-leaf` has not been set, in response to a `get-config` the server will return an empty container:

```
<rpc-reply>
  <data>
    <test-container/>
  </data>
</rpc-reply>
```

In commits prior to 6ab0617, the empty container was suppressed.

I've modified one of the test models in https://github.com/ADTRAN/Netopeer2/commit/b6d791099c1db6e33bb6f3a722e6ec8d14d9a439 to demonstrate the failure. The test `test_edit_delete1()` in `test_edit_get_config` will fail due to the unexpected empty `test-container`. The test will pass against older commits.

**Comment by michalvasko** [comment]
Hi,
you are right, sorry about that and it should work now. I have accidentally merged your test case, as well, but no harm done.

Regards,
Michal

**Comment by alangefe** [comment]
Hi Michal,

Your changes in a9844e2 pointed me in the right direction. They did need an additional change to handle default leafs more than one container deep. I've fixed that in #319, and added additional containers in the test model to verify the fix.

Thanks,
Andrew

**Comment by alangefe** [comment]
Fixed with #319, included in server-0.6.8.
